adoptopenjdk-api-v3-frontend/pom.xml

```diff
@@ -279,21 +246,6 @@
             </activation>
             <build>
                 <plugins>
-                    <plugin>
```

Comment:
Reworked to current default native build configuration, but... native build isn't working anyhow, should it?

Reply:
As far as I know, nobody is using it, so I am fine with removing support until someone asks for it.